tools/ovc/openvino/tools/ovc/main.py

```diff
@@ -11,6 +11,13 @@
 from openvino.tools.ovc.convert_impl import _convert
 from openvino.tools.ovc.cli_parser import get_model_name_from_args
 
+try:
+    # TODO 131000: temporal workaround to patch OpenVINO Core and frontends with tokenizers extensions
+    # make OVC tool to convert models requiring openvino-tokenizers extensions
+    import openvino_tokenizers
```

Comment:
This main.py is used for OVC cli tool only. Will it work for ov.convert_model()?
Maybe move to https://github.com/openvinotoolkit/openvino/blob/master/tools/ovc/openvino/tools/ovc/convert_impl.py ?

Reply:
The idea is only to internally import for ovc tool.
Now we won't have it for `convert_model` because `convert_model` and `read_model` logics get misaligned since `read_model` does not import `openvino_tokenizers`.
The right solution will be install openvino-tokenizers extensions to the right place of openvino and openvino Core will load these extensions itself. So it will be responsibility of openvino instead of openvino-tokenizers. It will be handled in CVS-131000. After that, I will remove this WA